torch/_C/__init__.pyi.in

```diff
@@ -2263,16 +2263,16 @@ Stack = List[IValue]
 
 class JitType:
     annotation_str: str
-    def isSubtypeOf(self, other: JitType) -> _bool: ...
-    def with_dtype(self, dtype: _dtype) -> JitType: ...
-    def with_sizes(self, sizes: List[Optional[_int]]) -> JitType: ...
+    def isSubtypeOf(self, other: Self) -> _bool: ...
```

Comment:
This sounds wrong? subtype check can get things that is not of this type?

Reply:
Should be Any